src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java

```diff
@@ -435,10 +437,11 @@ public void syncDrafts_fromCategoriesWithoutKeys_ShouldNotUpdateCategories() {
 
         assertThat(syncStatistics).hasValues(2, 0, 0, 2, 0);
         assertThat(callBackErrorResponses).hasSize(2);
-        assertThat(callBackErrorResponses.get(0))
-            .containsPattern("A duplicate value '.*' exists for field 'slug\\.en' on ");
-        assertThat(callBackErrorResponses.get(1))
-            .containsPattern("A duplicate value '.*' exists for field 'slug\\.en' on ");
+        assertThat(callBackErrorResponses.get(0)).contains("\"code\" : \"DuplicateField\"");
```

Comment:
👍

Reply:
Because sometimes I want to assert on the actual message string that is in the error call back message. However, you are right, sometimes it's good to assert also on the exception in the callback. I did that here where possible: 81f4437